### DISCUSSION TITLE
Add paste/.etc menu when holding textbox

### DISCUSSION THREAD

**Discussion by nyaruku** [ppy/osu#20482] [Ideas] [answered]
![Screenshot_20220926_083852_sh ppy osulazer](https://user-images.githubusercontent.com/38096703/192209982-2fa7bbb5-dffa-4154-a4a4-8de87d40a622.jpg)
Useful for ass long passwords

**Comment by peppy** [comment, marked as answer]
If on iOS you can use a three-finger outwards motion to paste. I'm not sure if there's similar on android.

We can definitely look at adding a context menu on textboxes. This will also depend on https://github.com/ppy/osu-framework/pull/5255 for it to work on mobile.
